test/CadlRanchProjects/authentication/http/custom/src/Custom.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Project name and solution name are too common. It would be better to change to `AuthenticationHttpCustom`.

Reply:
updated